### ISSUE TITLE
bisenetv2 : Questions about data augment

### ISSUE THREAD

**Issue by CallMeEricChiu** [CoinCheung/BiSeNet#65]
`trans_func(im_lb)` has `T.ColorJitter(brightness=0.4,contrast=0.4,saturation=0.4)`
wouldn't it change the pixel value of label after applying 'trans_func()' on label??
`   class CityScapes(Dataset):
    
    def __init__(self, datapth, trans_func=None, mode='train'):
        ######
    def __getitem__(self, idx):
        fn  = self.imnames[idx]
        impth, lbpth = self.imgs[fn], self.labels[fn]
        img, label = cv2.imread(impth), cv2.imread(lbpth, 0)
        label = self.lb_map[label]
        im_lb = dict(im=img, lb=label)
        if not self.trans_func is None:
            im_lb = self.trans_func(im_lb)
        im_lb = self.to_tensor(im_lb)
        img, label = im_lb['im'], im_lb['lb']
        return img.detach(), label.unsqueeze(0).detach()

    def __len__(self):
        return self.len`

`class TransformationTrain(object):

    def __init__(self):
        self.trans_func = T.Compose([
            T.RandomResizedCrop([0.375, 1.], [512, 1024]),
            T.RandomHorizontalFlip(),
            T.ColorJitter(
                brightness=0.4,
                contrast=0.4,
                saturation=0.4
            ),
        ])

    def __call__(self, im_lb):
        im_lb = self.trans_func(im_lb)
        return im_lb`

**Comment by alphashuo** [comment]
take a look at the specific \_\_call\_\_ func of ColorJitter. 
Cause it does not change the shape of img, so we do not need to apply the transform on the label.

https://github.com/CoinCheung/BiSeNet/blob/81a88852fa50eee96738d72f4dbfa2de108bf3e9/bisenetv2/transform_cv2.py#L82-L94
